spec/swagger.yaml

```diff
@@ -82,6 +82,10 @@ tags:
           }
       }
 
+  - name: "Account & Projects"
+    description: |-
+      Each client within Smartling is given their own Account with a designated accountUid. Within each account is any number of Projects. All files, content, jobs are tied to a particular Project. The Accounts & Projects API will allow you to programmaticly find and use the Projects within your Account.
```

Comment:
Each client within Smartling is given their own account with a designated `accountUid`. Within each account is any number of projects. All files, content, and jobs are tied to a particular project. The Accounts & Projects API will allow you to programmatically find and use the projects within your account.

Reply:
applied